lib/web/apiserver.go

```diff
@@ -909,7 +909,8 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Channel can contain "/", hence the use of a catch-all parameter
 	h.GET("/webapi/automaticupgrades/channel/*request", h.WithUnauthenticatedHighLimiter(h.automaticUpgrades))
 
-	// Create Machine ID bots
+	// Machine ID bots
+	h.GET("/webapi/sites/:site/machine-id/bot", h.WithClusterAuth(h.getBot))
```

Comment:
Should this be `/webapi/sites/:site/machine-id/bot/:name` so we don't conflict with a list endpoint?